packages/node/src/store.ts

```diff
@@ -45,13 +45,10 @@ export class Store {
       `${this.storeKeyPrefix}/${DB_NAMESPACE_CHANNEL}`
     )) as { [multisigAddress: string]: StateChannelJSON };
 
-    if (!channelsJSON) {
-      console.log("No channels exist yet");
-    } else {
-      for (const entry of Object.entries(channelsJSON)) {
-        channels[entry[0]] = StateChannel.fromJson(entry[1]);
-      }
+    for (const entry of Object.entries(channelsJSON || {})) {
```

Comment:
@snario We can avoid using the sub-index accessors by destructuring directly on the iterator:

```ts
for (const [key, value] of Object.entries(channelsJSON)) {
  channels[key] = StateChannel.fromJson(value);
}
```

Also, it might be best to add the `|| {}` fallback directly in the declaration for `channelsJSON`.